### PR TITLE
#168055387 Fix page and limit validation on users can view all authors route

### PR DESCRIPTION
**Pull request by kennyedward**
#### What does this PR do?
This pull request implements paginator middleware on getAllAuthors endpoint

#### Description of Task to be completed
- Valid user can view all available authors

#### How should this be manually tested?
- Clone this repository
- ```cd into the repo``` and run ```npm install```
- Using Postman *SET* ```Authorization``` to ```Bearer <user token>```
- Make a GET request to /api/v1/authors

#### Any Background Context You want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#168055387](https://pivotaltracker.com/story/show/168055387)

#### Screenshots (if appropriate)
All Authors

#### Questions:
N/A

### CODE REVIEW

**Comment by iverenshaguy** [comment]
Please update you commit scope to something like  `validations` not `page validator`

File: .env.example [hunk 1]
```diff
@@ -13,8 +13,7 @@ SESSION_SECRET=
 SENDGRID_API_KEY=
 SENDGRID_EMAIL=
 SENDGRID_ACTIVATION_TEMPLATE=
-SENDGRID_ACTIVITY_TEMPLATE=
```
Comment (iverenshaguy) [inline .env.example]: Replace all these

**Review by iverenshaguy — CHANGES_REQUESTED** [review]
Good job, please update the PR based on my comments

File: .env.example (hunk 1 above)
Comment (kennyedward) [reply .env.example]: Alright. Fixed. I also used the opportunity to add more tests to bookmark, update and reset password

**Comment by kennyedward** [comment]
> Please update you commit scope to something like `validations` not `page validator`

Alright.

File: .env.example (hunk 1 above)
Comment (iverenshaguy) [reply .env.example]: Thanks

**Review by iverenshaguy — APPROVED** [review]

**Pull request merged**